twitter/TweetCell.m

```diff
@@ -21,4 +23,44 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     // Configure the view for the selected state
 }
 
+- (IBAction)didTapFavorite:(id)sender {
+    // Update the local tweet model
+    if (self.tweet.favorited == NO) {
+        self.tweet.favorited = YES;
+        self.tweet.favoriteCount += 1;
+        // Send a POST request to the POST favorites/create endpoint
+        [[APIManager shared] favorite:self.tweet completion:^(Tweet *tweet, NSError *error) {
+            if(error){
+                 NSLog(@"Error favoriting tweet: %@", error.localizedDescription);
+            }
+            else{
+                NSLog(@"Successfully favorited the following Tweet: %@", tweet.text);
+            }
+        }];
+        // Change the image
+        [sender setImage:[UIImage imageNamed:@"favor-icon-red"] forState:UIControlStateNormal];
+    }
+    else {
+        self.tweet.favorited = NO;
+        self.tweet.favoriteCount -= 1;
+        // Send a POST request to the POST favorites/create endpoint
+        [[APIManager shared] unfavorite:self.tweet completion:^(Tweet *tweet, NSError *error) {
+            if(error){
```

Comment:
nitpicky (in our larger code bases we might rely on auto-formatting and lint rules)

let's try and keep the spacing consistent here (spaces between control statements and parenthesis, brackets etc)